README.md

```diff
@@ -203,13 +203,13 @@ We support DevContainers for VSCode. [Click here to get started.](https://vscode
 
 ## Docker
 
-🚧 Docker containers and images are current in progress. We are actively working on bringing a simple docker build and publish pipeline for Documenso.
+🚧 Docker containers and images are currently in progress. We are actively working on bringing a simple docker build and publish pipeline for Documenso.
 
 ## Self Hosting
 
 We support a variety of deployment methods, and are actively working on adding more. Stay tuned for updates!
 
-> Please note the below deployment methods are for v0.9, we will update these to v1.0 once it has been released.
+> Please note the below deployment methods are for v0.9, we will update them to v1.0 once it has been released.
```

Comment:
Just a suggestion, but maybe `are for v0.9, **and** we will update` will sound better.